src/NHibernate.Validator/Engine/IClassValidator.cs

```diff
@@ -50,6 +50,12 @@ public interface IClassValidator
 		/// <param name="persistentClass">hibernate metadata</param>
 		void Apply(PersistentClass persistentClass);
 
+		/// <summary>
+		/// Create validators based on hibernate metadata if appropriative validator not defined explicitly
+		/// </summary>
+		/// <param name="properties">Hibernate metadata to analize</param>

```

Comment:
Same troubles on this xml documentation than on its concrete implementation counterpart.

Reply:
Fixed